src/stdatamodels/jwst/_kwtool/compare.py

```diff
@@ -65,6 +65,20 @@ def __repr__(self):
             },
         },
     },
+    ("PRIMARY", "FOCUSPOS"): {
+        "type": {
+            "dmd": {
+                "union": {"integer"},
+            },
+        },
+    },
+    ("PRIMARY", "MRSPRCHN"): {
+        "enum": {
+            "dmd": {
+                "difference": {"ALL"},
```

Comment:
I wasn't quite sure about this one but the datamodel schemas list "ALL" but the keyword dictionary does not.